scripts/benchmarks/benches/compute.py

```diff
@@ -175,7 +210,7 @@ def __init__(self, bench, ioq, isCopyOnly, source, destination, size):
         self.source = source
         self.destination = destination
         self.size = size
-        super().__init__(bench, "api_overhead_benchmark_sycl", "ExecImmediateCopyQueue")
+        super().__init__(bench, "api_overhead_benchmark_sycl", "ExecImmediateCopyQueue", "time [us]")
```

Comment:
You've set a default unit, do you need to set these?

Reply:
Not longer needed.